test/TestCases.test.js

```diff
@@ -60,20 +60,37 @@ describe('TestCases', () => {
             return;
           }
           const expectedDirectory = path.resolve(directoryForCase, 'expected');
-          for (const file of fs.readdirSync(expectedDirectory)) {
-            const content = fs.readFileSync(
-              path.resolve(expectedDirectory, file),
-              'utf-8'
-            );
-            const actualContent = fs.readFileSync(
-              path.resolve(outputDirectoryForCase, file),
-              'utf-8'
+
+          for (const file of walkSync(expectedDirectory)) {
+            const actualFilePath = file.replace(
+              new RegExp(`/cases/${directory}/expected/`),
```

Comment:
This doesnt't work on windows, or if directory contains special chars.

Reply:
If it fails to replace it will compare the file with itself making the test always pass.

Reply:
Maybe use `path.relative` instead.

Reply:
good catch, fixed. I used `path.resolve` instead though because the output file can be in a nested directory.

Reply:
@sokra Just pushed an update, can you verify the `actualFilepath` works on windows now?

Reply:
now using `path` to correctly resolve the directories